scenario/urls.py

```diff
@@ -9,4 +9,7 @@
     path('<int:pk>/set_starter_module/<int:module_id>/', viewsets.scenario_starter_module),
     path('<int:pk>/schedule/', viewsets.scenario_schedule),
     path('all_modules/<int:scenario_id>', viewsets.all_modules_scenario),
+    path('execute/<int:scenario_id>', viewsets.execute),
```

Comment:
ye kari kon con url injuri hast ke "{IP_ADDRESS}/api/scenario/", bad e in scenario e id ish biad bad bezanim execute masalan behtare be nazaram, hala mishe bahs kard